target/linux/ipq806x/files-5.4/arch/arm/boot/dts/qcom-ipq8064-g10.dts

```diff
@@ -0,0 +1,366 @@
+// SPDX-License-Identifier: GPL-2.0
+#include "qcom-ipq8064-v1.0.dtsi"
+
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/soc/qcom,tcsr.h>
+
+/ {
+	compatible = "asrock,g10", "qcom,ipq8064";
+	model = "ASRock G10";
+
+	aliases {
+		mdio-gpio0 = &mdio_gpio;
+		ethernet0 = &gmac1;
+		ethernet1 = &gmac0;
+
+		led-boot = &led_blue;
+		led-failsafe = &led_fail;
+		led-running = &led_blue;
+		led-upgrade = &led_blue;
+	};
+
+	reserved-memory {
+		#address-cells = <1>;
+		#size-cells = <1>;
+		ranges;
+
+		rsvd@41200000 {
+			reg = <0x41200000 0x300000>;
+			no-map;
+		};
+	};
+
+	chosen {
+		stdout-path = "serial0:115200n8";
```

Comment:
This has been moved to parent dtsi in https://github.com/openwrt/openwrt/commit/74e339f118e600f2169965989ec3618a39e69759